tests/e2e/tests/misc/proxy-config.ts

```diff
@@ -0,0 +1,59 @@
+import * as express from 'express';
```

Comment:
Can you rename the `proxy.ts` test to `proxy-config.ts`, and do all tests in a single file using the `--proxy-config` flag? I think we must have renamed the flag at one point and forgot to change the test name.